e2e/atlas/helper_test.go

```diff
@@ -128,6 +129,40 @@ func deployCluster() (string, error) {
 	return clusterName, nil
 }
 
+func newAvailableRegion(projectID, tier, provider string) (string, error) {
```

Comment:
[q] why pass the project ID? shouldn't it use the env as well?

Reply:
yes, this is a leftover from my previous approach with the store